gfx.cpp

```diff
@@ -121,6 +121,7 @@ class GfxInternal
     std::vector<D3D12_RESOURCE_BARRIER> resource_barriers_;
     ID3D12Resource **back_buffers_ = nullptr;
     D3D12MA::Allocation **back_buffer_allocations_ = nullptr;
+    DXGI_FORMAT back_buffer_format_ = DXGI_FORMAT_R8G8B8A8_UNORM;
```

Comment:
Made this into a member variable so we don't have to re-fetch that `DXGI_SWAP_CHAIN_DESC1` structure each time.